octoprint_mrbeam/static/js/settings/custom_material.js

```diff
@@ -20,6 +20,19 @@ $(function () {
                 self.readLocalFile(ev, function (data) {
                     if (data && "custom_materials" in data) {
                         console.log("Loaded material settings: ", data);
+                        //  Update custom material settings exported with new keys and laser model value
+                        for (let [material_id, material_settings] of Object.entries(data.custom_materials)) {
+                            if ('laser_type' in material_settings &&
+                                material_settings['laser_type'] === "MrBeamII-1.0") {
+                                material_settings['laser_type'] = '0';
+                                material_settings['laser_model'] = material_settings['laser_type'];
+                                delete material_settings['laser_type'];
```

Comment:
See previous comment. No need to re-write a value if you are going to delete it.

Also - I had hoped that the sanitisation function would be the same for the front and backend (By same I mean that it is the same piece of code, not a mirror of one-another)